.github/CHANGELOG.md

```diff
@@ -2,6 +2,45 @@
 
 <h3>New features since last release</h3>
 
+<h4>PennyLane now comes packaged with PennyLane-Lightning</h4>
+
+* The C++-based [PennyLane-Lightning](https://pennylane-lightning.readthedocs.io/en/latest/) device
+  is now included with installations of PennyLane. The PennyLane-Lightning device is a fast
+  state-vector simulator equipped with the efficient
+  [adjoint method](https://arxiv.org/abs/2009.02823) for differentiating quantum circuits. The
+  device can be accessed using the `"lightning.qubit"` name:
+  
+  ```python
+  import pennylane as qml
+
+  wires = 3
+  layers = 2
+  dev = qml.device("lightning.qubit", wires=wires)
+
+  @qml.qnode(dev, diff_method="adjoint")
```

Comment:
do you want to mention in the text above the native adjoint support 🙂 It is quite a big feature!

Reply:
Definitely! Have updated to emphasize more: 165a43819d5f732f068c9b24af04c22b55f304cb